### ISSUE TITLE
Provide readable source code?

### ISSUE THREAD

**Issue by chrimage** [ozenc/LG-R105-Camera-App#1]
This project is great! I was able to get it running on my LG-R105 quite easily. Can you help me understand what language was used to create this app? The unreadable HTML and JS code without line breaks suggests that maybe you used TypeScript or Vue to produce this application? I am new to modern web development, I hope that question made sense.

**Comment by ozenc** [comment]
Hey, it's great that you liked it. Sorry for late answer. I am new to these  modern web applications also. Basically I used Vue+vuetify and simple javascript  to utilize API on the camera. I wanted this repository more practical for whom is just wanted to use their cameras as fast as possible and pain free. When I have time to cleanup my code I would love to share of course. 
Our camera is using google OSC api inside. (https://developers.google.com/streetview/open-spherical-camera?hl=en)
Basically you can call api for getting and setting things.  For now I'm attaching my basic code for interacting with the camera in html and vanilla-js. Hope this helps. 

[index.html.txt](https://github.com/ozenc/LG-R105-Camera-App/files/10803576/index.html.txt)
